src/utils/hwWallet.js

```diff
@@ -10,37 +10,57 @@ import { getTransactionBytes, calculateTxId, getBufferToHex } from './rawTransac
 import { PLATFORM_TYPES, getPlatformType } from './platform';
 import store from '../store';
 import loginTypes from '../constants/loginTypes';
+import { accountLoggedOut } from '../actions/account';
+import { dialogDisplayed, dialogHidden } from '../actions/dialog';
+import Alert from '../components/dialog/alert';
 
 const util = require('util');
 
 const { ipc } = window;
 
+const handleConnect = () => {
+  store.dispatch({
+    type: actionTypes.settingsUpdated,
+    data: { isHarwareWalletConnected: true },
+  });
+  store.dispatch(errorToastDisplayed({ label: HW_MSG.LEDGER_CONNECTED }));
```

Comment:
here it says `HW_MSG.LEDGER_CONNECTED`, but this function `handleConnect` is used also for `ipc.on('trezorConnected', handleConnect);` below.

Reply:
fixed